libs/community/tests/unit_tests/vectorstores/test_sqlserver.py

```diff
@@ -584,7 +561,11 @@ def test_that_given_a_connection_string_with_uid_and_pwd_entra_id_auth_is_not_us
 @mock.patch(
     "langchain_community.vectorstores.sqlserver.SQLServer_VectorStore._provide_token"
 )
+@mock.patch(
+    "langchain_community.vectorstores.sqlserver.SQLServer_VectorStore._prepare_json_data_type"
+)
 def test_that_connection_string_with_trusted_connection_yes_does_not_use_entra_id_auth(
+    prep_data_type: Mock,
     provide_token: Mock,
     dialect_initialize: Mock,
 ) -> None:
```

Comment:
General comment: I notice, the base branch of your fork has fallen way behind the community_package main, might be useful to bring it upto speed soon.